cmd/juju/application/config.go

```diff
@@ -291,7 +298,7 @@ func (c *configCommand) getConfig(client ApplicationAPI, ctx *cmd.Context) error
 	}
 
 	logger.Infof("format %v is ignored", c.out.Name())
-	key := c.configBase.KeyToGet
+	key := c.configBase.KeysToGet[0]
```

Comment:
Have we already checked that len(KeysToGet) > 0? It appears we have below. I wonder if a check here is should be done (again) just in case - an error is better than a panic.

Reply:
`config/config.go` has the following:
```go
if len(c.KeysToGet) > 0 {
	c.Actions = append(c.Actions, GetOne)
}
```
This is the only place where `GetOne` is added to `Actions`. Is this safe enough, or should I still add a check elsewhere?

Reply:
Yeah, I saw the code. At the least, a comment saying that we can only get here if KeysToGet is not empty is good.
Although, the fear is that a programming error later could cause a panic, and so a defensive check returning an error is probably be worthwhile. It all works now but may not later if someone refactors and doesn't realise because the point at which KetsToGet is used is away from where it is set up.

Reply:
Ok will put that in